apps/daffio/src/app/docs/components/heading/heading.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  Component,
+  ElementRef,
+  Input,
+  AfterViewInit,
+  ChangeDetectionStrategy,
+} from '@angular/core';
+
+@Component({
+  selector: 'daffio-ce-heading',
+  templateUrl: './heading.component.html',
+  styleUrls: ['./heading.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MarkdownHeadingComponent implements AfterViewInit {
+  @Input() size = 'h1';
+  @Input() slug = '';
+
+  iconColor = 'dimgrey';
+
+  constructor(private el: ElementRef) {}
+
+  ngAfterViewInit() {
+    const heading = this.el.nativeElement.querySelector('h1, h2, h3, h4, h5, h6');
+    if (heading) {
+      heading.classList.add('daffio-ce-heading');
+    }
```

Comment:
this is not hydration safe. What are you trying to accomplish here? This class doesn't have any styles associated with it.